### PR TITLE
So fixes main

### PR DESCRIPTION
**Pull request by suryaoruganti**

### CODE REVIEW

File: modules/aws/rds/variables.tf [hunk 1]
```diff
@@ -54,17 +70,19 @@ variable "identifier" {
...
 variable "vpc" {
   description = "All vpc info"
   type = object({
     name = string
-    id   = string
+    vpc_id   = string
     public_subnets = list(string)
     private_subnets = list(string)
-    database_subnets_cidr_blocks = list(string)
+    database_subnets = list(string)
```
Comment (tiny-argonaut) [inline modules/aws/rds/variables.tf:84]: @suryaoruganti We are not using below variables in `main.tf` file of rds module.
- database_subnets
- default_security_group_id

File: modules/aws/rds/main.tf [hunk 2]
```diff
@@ -45,35 +39,43 @@ module "db" {
...
-  performance_insights_enabled          = true
-  performance_insights_retention_period = 7
+
   port                                  = 5432
   publicly_accessible                   = true
   storage_encrypted                     = true
   storage_type                          = "gp2"
-  vpc_security_group_ids                = [var.vpc.default_security_group_id]
+
+  performance_insights_enabled          = true
```
Comment (admiralnico) [inline modules/aws/rds/main.tf:77]: do you want this to be enabled by default? dont think a lot of users will use it initially. Thoughts?

File: modules/aws/rds/variables.tf (hunk 1 above)
Comment (suryaoruganti) [reply modules/aws/rds/variables.tf:84]: Ack, but not an issue

File: modules/aws/rds/main.tf (hunk 2 above)
Comment (suryaoruganti) [reply modules/aws/rds/main.tf:77]: thought it was a simple add on and free for a 7 day rolling period. Is there a reason to not enable it?

Comment (admiralnico) [reply modules/aws/rds/main.tf:77]: I missed the 7 day part..for 7 days it should be okay..

Comment (suryaoruganti) [reply modules/aws/rds/main.tf:77]: 👍  I can merge if you approve. Want to set a tag to main and test out stuff

**Review by admiralnico — APPROVED** [review]
LGTM

**Pull request merged**